services/movie.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Genre, Actor, Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> Movie:
```

Comment:
The return type hint for `get_movies` should be `QuerySet[Movie]` instead of `Movie`, as the function returns a QuerySet, not a single instance.